### PR TITLE
Allow ports to be configured as `NodePort`

### PR DESCRIPTION
**Pull request by plblueraven**
This pull request allows ports to be configured as a user-defined `NodePort`. This is useful when using this helm chart in local development, e.g. on k3s.

### CODE REVIEW

**Comment by SamirPS** [comment]
In case of the else do we still need to put the Nodeport in the config since it's null?

**Review by bashofmann — DISMISSED** [review]
Can you remove the changelog entries and the version bump in Chart.yaml.

Right now this would automatically create a release. We will take care of creating the changelog and creating a release separately.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```
[the remark this answers is not in the document]
Comment (plblueraven) [reply CHANGELOG.md]: OK, done. I hope I have understood you correctly.

File: charts/qdrant/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
```
Comment (SamirPS) [inline charts/qdrant/CHANGELOG.md:3]: Sorry also here and after it's good

**Review by SamirPS — CHANGES_REQUESTED** [review]

File: charts/qdrant/CHANGELOG.md (hunk 2 above)
Comment (plblueraven) [reply charts/qdrant/CHANGELOG.md]: No problem. Done.

Comment (SamirPS) [reply charts/qdrant/CHANGELOG.md]: Thank

**Pull request merged**